src/main/java/com/sap/cap/cds/intellij/lsp/ServerDescriptor.java

```diff
@@ -24,6 +25,15 @@ public class ServerDescriptor extends ProjectWideLspServerDescriptor {
 
     public ServerDescriptor(@NotNull Project project, @NotNull String presentableName) {
         super(project, presentableName);
+        validateNodeExecutable();
+    }
+
+    private void validateNodeExecutable() {
+        try {
+            new GeneralCommandLine("node", "-e", "").createProcess();
```

Comment:
We should compare the result with the minimum required version in cds-lsp/package.json

Reply:
Right, this is a good idea. Currently, on an outdated Node, the IDE shows the CDS LSP server as 'Initializing' for a long time, then as 'Terminated' (with env var `DEBUG=cds-lsp`).

Reply:
Done. Since cds-lsp is most competent to ensure the right Node.js version, https://github.com/cap-js/cds-intellij/pull/27/commits/edb163eaf9332a56ec70ef254f798f0c673610ab delegates this decision to the server.